### ISSUE TITLE
Reading: How Does Fine-tuning Affect the Geometry of Embedding Space: A Case Study on Isotropy

### ISSUE THREAD

**Issue by a1da4** [a1da4/paper-survey#201]
## 0. Paper
paper: [link](https://aclanthology.org/2021.findings-emnlp.261/)

## 1. What is it?
They analyzed the isotropy in fine-tuned contextual word representations.

## 2. What is amazing compared to previous works?
Previous works show that the contextual word representations are anisotropy.
In this paper, they focus on the geometric analysis of the fine-tuned contextual word representations.

## 3. Where is the key to technologies and techniques?
They try to answer the following three questions:
1. Is there any relation between isotropy and improved performance by fine-tuning?
2. Isotropy is the same positive outcome for fine-tuning?
3. How does the distribution change upon fine-tuning?

## 4. How did evaluate it?
- task: Semantic Textual Similarity (label is a continuous range in 0 to 5)
- models: BERT, RoBERTa
- method:
  - Zero-mean
  - Clustering+ZM: conduct Zero-mean in each cluster
  - Global app.: all-but-the-top. Zero-mean + removing top dominant directions
  - Cluster-based app.: Clustering+ZM + removing top dominant directions

### 4.1 (Q.1, Q.2) Fine-tuning does not contribute to isotropy 
- pre-trained: performance↓, isotropy↑
- fine-tuning: performance↑, isotropy↓

![スクリーンショット 2021-09-13 23 49 05](https://user-images.githubusercontent.com/45454055/133105730-15a0a5a3-9eab-4482-9e24-4890463afe0b.png)

### 4.2 (Q.3) Fine-tuning removes frequency bias
- pre-trained: there are local areas that encode frequency information
- fine-tuned: those areas are removed
![スクリーンショット 2021-09-13 23 52 25](https://user-images.githubusercontent.com/45454055/133106312-bf45caba-e339-4d8b-8e67-ecdfa8d168d5.png)
